web_app_drawer/static/src/css/web_app_drawer.css

```diff
@@ -32,10 +32,12 @@ main {
     border-radius: 50%;
 }
 
-#odooMenuBarNav[aria-expanded="false"] {
-    /* Hack to hide the visibly expanded mobile menu on load. */
-    position: absolute;
-    z-index: -9999;
+@media (max-width: 768px) {
```

Comment:
Transform this file to LESS and use [`@screen-xs-max`](https://github.com/twbs/bootstrap/blob/v3.3.6/less/variables.less#L315), please.